includes/class-wc-cardknox-applepay-blocks-support.php

```diff
@@ -23,23 +23,30 @@ public function get_payment_method_script_handles() {
         $handle     = 'wc-cardknox-blocks';
         $base_dir   = plugin_dir_path(__FILE__) . '../blocks/build/';
         $asset_file = $base_dir . 'index.asset.php';
-        $script_url = plugins_url('../blocks/build/index.js', __FILE__);
-
-        $asset = file_exists($asset_file)
-            ? include $asset_file
-            : [
-                'dependencies' => [ 'wp-element', 'wc-blocks-registry', 'wc-blocks-checkout' ],
-                'version'      => file_exists($base_dir . 'index.js') ? filemtime($base_dir . 'index.js') : time(),
-            ];
-
+        $script_url = plugin_dir_url(__FILE__) . '../blocks/build/index.js';
+    
+        
+        $asset = [
+            'dependencies' => [ 'wp-element', 'wc-blocks-registry', 'wc-blocks-checkout' ],
+            'version'      => file_exists($base_dir.'index.js') ? filemtime($base_dir.'index.js') : time(),
+        ];
+    
+        if ( file_exists($asset_file) ) {
+            $loaded = include_once $asset_file;     // NOSONAR php:S2006
```

Comment:
Can you explain this comment?
Is `include` or `include_once` the recommended approach?